scripts/deployment/deploy_06_gnosis_safe_multisig.js

```diff
@@ -27,27 +27,41 @@ async function main() {
     const deployer = await EOA.getAddress();
     console.log("EOA is:", deployer);
 
+    // Multisig implementation contract parameters supplied via multisigImplementation.json file
+    const multisigDataFromJSON = fs.readFileSync("multisigImplementation.json", "utf8");
+    const parsedMultisigData = JSON.parse(multisigDataFromJSON);
+    let multisigImplementationContractName = parsedMultisigData.multisigContractName;
+
     // Transaction signing and execution
-    console.log("6. EOA to deploy GnosisSafeMultisig");
-    const GnosisSafeMultisig = await ethers.getContractFactory("GnosisSafeMultisig");
-    console.log("You are signing the following transaction: GnosisSafeMultisig.connect(EOA).deploy()");
-    const gnosisSafeMultisig = await GnosisSafeMultisig.connect(EOA).deploy(gnosisSafeAddress,
-        gnosisSafeProxyFactoryAddress);
+    console.log("6. EOA to deploy " + multisigImplementationContractName);
+    const GnosisSafeMultisig = await ethers.getContractFactory(multisigImplementationContractName);
+    console.log("You are signing the following transaction: " + multisigImplementationContractName + ".connect(EOA).deploy()");
+
+    // Verify the construct parameters and add them here, if needed
+    const gnosisSafeMultisig = await GnosisSafeMultisig.connect(EOA).deploy();
```

Comment:
Deploy goes without arguments by default. If there are no arguments, then it will go through, if the are arguments and they are not set, the call to `deploy()` will give the message about missing arguments